src/plugins/workspace/server/saved_objects/workspace_ui_settings_client_wrapper.ts

```diff
@@ -135,11 +119,7 @@ export class WorkspaceUiSettingsClientWrapper {
           options
         );
 
-        if (workspaceUpdateResult.attributes.uiSettings) {
-          configObject.attributes = workspaceUpdateResult.attributes.uiSettings;
-        }
-
-        return configObject as SavedObjectsUpdateResponse<T>;
+        return workspaceUpdateResult.attributes.uiSettings as SavedObjectsUpdateResponse<T>;
```

Comment:
Still, is it able to remove the type cast here? And the value seems not consistent, previous `configObject: { attributes: finalSettings }`, not the result is the finalSettings itself.